features/graph/documentation/api-authentication.md

```diff
@@ -0,0 +1,116 @@
+# Netlify Graph - API Authentication
+
+> **NOTE:** API Authentication is a **Beta** feature released through [Netlify Labs](https://www.netlify.com/blog/2021/03/31/test-drive-netlify-beta-features-with-netlify-labs/). Its functionality is subject to change. We strongly recommend not using it in any production and/or critical workflows.
```

Comment:
Potentially remove this, depending on what you decide to do with my suggestion on the readme.